ansible-ipi-install/inventory/hosts

```diff
@@ -0,0 +1,11 @@
+[masters]
+master-0 name=master-0 role=master user=admin password=password ipmi_address=192.168.1.1 mac=ec:f4:bb:da:0c:58
```

Comment:
should user and password be ipmi_user and ipmi_password
shoudl mac be provison_mac